QgisModelBaker/ui/edit_command.ui

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>EditCommandDialog</class>
+ <widget class="QDialog" name="EditCommandDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>413</width>
+    <height>266</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Edit command</string>
+  </property>
+  <layout class="QGridLayout" name="gridLayout">
+   <item row="3" column="0" colspan="2">
+    <widget class="QDialogButtonBox" name="button_box">
+     <property name="orientation">
+      <enum>Qt::Horizontal</enum>
+     </property>
+     <property name="standardButtons">
+      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
+     </property>
+    </widget>
+   </item>
+   <item row="2" column="0" colspan="2">
+    <widget class="QPlainTextEdit" name="command_edit"/>
+   </item>
+   <item row="0" column="0" colspan="2">
+    <widget class="QLabel" name="label_warning">
+     <property name="text">
+      <string>There will be dragons...</string>
+     </property>
+    </widget>
+   </item>
+   <item row="1" column="0" colspan="2">
+    <widget class="QCheckBox" name="chk_activate">
+     <property name="text">
+      <string>No worries. I'm a fairy.</string>
```

Comment:
:rofl:  What is that for?

Reply:
Fairies are not afraid of dragons. At least most of them :-) It's the fill text I will change. See https://github.com/opengisch/QgisModelBaker/pull/424#issuecomment-667109717 and I see you already suggested a text for the dragons. Thanks...

Reply:
Ah, it's the "safety guard" check box?
Shall we leave it out (and hope that the "dragon text" is enough context information)?

Reply:
I'm open. I made it that you have to check to be able to edit the command. Otherwise it's just displayed.
You can decide.

Reply:
I was just thinking. This will only be usable by advanced users (you have to know about ili2db command line which requires being able to read some docs or do other internet research), those will also be smart enough to read the "dragon" instruction and click away the checkbox. So basically I think editing the command is safety measure enough.

Reply:
okay :fairy: